src/platform/native.zig

```diff
@@ -214,6 +238,23 @@ pub const Platform = struct {
         while (platform.events.popFirst()) |ev| {
             platform.allocator.destroy(ev);
         }
+
+        if (builtin.os.tag == .linux) {
```

Comment:
Maybe move these blocks into a separate function we call from here, e.g. `platform.deinitLinuxGameMode()`